html-forms/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Forms</title>
+  </head>
+  <body>
+    <h1> Coding Form! </h1>
+    <form action="" method="get">
+      <label for="email">Email <br> </label>
```

Comment:
for labels to work properly, you need to make sure the `for` attribute on the `label` matches the `id` attribute on the input. Or you need to wrap the input with the label. Both will have the same functionality.